equipment-datastore-rdbms/src/main/java/com/telecominfraproject/wlan/equipment/datastore/rdbms/EquipmentDAO.java

```diff
@@ -177,6 +177,9 @@
 
     private static final String SQL_GET_BY_CUSTOMER_ID_AND_EQUIPMENT_TYPE = "select " + ALL_COLUMNS + " from "
             + TABLE_NAME + " " + " where customerId = ? and equipmentType = ? ";
+    
+    private static final String SQL_GET_BY_CUSTOMER_ID_AND_SEARCH_BY_MAC_AND_NAME = "select " + ALL_COLUMNS + " from "
```

Comment:
Please add the following index to the equipment table in the schema-postgres.sql and schema-hsqldb-test.sql:

create index if not exists idx_equipment_customerId_name on equipment (customerId, name);

(index for the baseMacAddress already exists)